default.ps1

```diff
@@ -78,7 +78,7 @@ task Nuget {
       @("RhinoDSL", "1.0.0"), `
       @("Common.Logging", "3.0.0"), `
       @("Common.Logging.Core", "3.0.0"), `
-      @("FileHelpers", "2.0.0.0") `
+      @("FileHelpers", "3.0.46-beta") `
```

Comment:
@MarcosMeli if you could refresh the dll in the SharedLibs folder and update the default.ps1 in your branch I think your pull request will be more successful then mine. Any way if my pull request is accepted I've already prepared a "nugetified" version of the update as well on my branch that is mergeable as well.